contents/staticAnalysis/PMD.md

```diff
@@ -0,0 +1,73 @@
+## PMD
+PMD is a static analyser for Java, JavaScript, Salesforce.com Apex, PL/SQL, Apache Velocity, XML, XSL. The Copy-pate-detector ([CPD Find duplicated code](http://pmd.sourceforge.net/pmd-4.3.0/cpd.html)) is also included as an add-on in PMD.
```

Comment:
typo, should be copy/paste-detector. You should briefly mention what it is about.